index.bs

```diff
@@ -798,17 +802,33 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 		{{Clipboard/read()}} returns a {{Promise}} to [=clipboard items=] object that represents contents of [=system clipboard data=].
 		</p>
 
+		An <dfn>unsanitized</dfn> object is a [=sequence=] of {{DOMString}}s corresponding to the [=representation/mime type=]. It contains <dfn>unsanitized mime types</dfn> that is currently limited to "text/html".
+
+		<p class="note">
+		In the future, other MIME types may be added.
+		</p>
+
 		The <dfn>clipboard task source</dfn> is triggered in response to reading or writing of [=system clipboard data=].
 
 		<div id="clipboard-idl" dfn-for="Clipboard">
 		<div class="algorithm" data-algorithm="clipboard-read">
-		<h4 method for="Clipboard">read()</h4>
-		The {{Clipboard/read()}} method must run these steps:
+		<h4 method for="Clipboard">read(|formats|)</h4>
+		The {{Clipboard/read(formats)}} method must run these steps:
 
 			1. Let |realm| be [=this=]'s [=relevant realm=].
 
 			1. Let |p| be [=a new promise=] in |realm|.
 
+			1. Let |format| be a {{DOMString}}.
```

Comment:
Shouldn't |formats| be a dictionary instead of just a DOMString?

Reply:
Yep, `|formats` is a dictionary, this variable just stores the first DOMString from that dictionary in the below step:
`Set |format| to |formats|[0].`

Reply:
but you would be iterating over `formats.unsanitized`, rather than `formats`, no?

Reply:
Ah you're right. Missed that part.

Reply:
Done.